app/views/start_page.scala.html

```diff
@@ -14,77 +14,73 @@
+@reportLink = {<a class="govuk-link" href="@{appConfig.serviceAvailabilityUrl}">@messages("startPage.problemsWithServiceNotice.link")</a>}
+@beforeYouStartLink = {<a class="govuk-link" href=@{appConfig.customsDeclarationsGoodsTakenOutOfEuUrl}>@messages("startPage.beforeYouStart.line.1.link")</a>}
+@govukLayout(title = Title("startPage.title", Some("startPage.title.sectionHeader"))) {
 
-  <div class="grid-row">
-    <div class="column-full">
-      @components.heading(
-        sectionHeader = messages("startPage.title.sectionHeader"),
-        title = messages("startPage.title")
-      )
-    </div>
-    <div class="column-full">
-      <p id="description" class="lede">@messages("startPage.description")</p>
-    </div>
-  </div>
-
-  <div class="grid-row">
-    <div class="column-full">
-
-      <p id="contents">@messages("startPage.contents.header")</p>
-      <ul id="contents-list" class="dashed-list">
-        <li class="dashed-list-item"><a href="#before-you-start">@messages("startPage.beforeYouStart.header")</a></li>
-        <li class="dashed-list-item"><a href="#information-you-need">@messages("startPage.informationYouNeed.header")</a></li>
-        <li class="dashed-list-item"><a href="#report-your-arrival-and-departure">@messages("startPage.reportYourArrivalAndDeparture.header")</a></li>
-      </ul>
-
-      <h2 id="before-you-start" class="heading-medium">@messages("startPage.beforeYouStart.header")</h2>
-      <p id="before-you-start-element-1">
-        @messages("startPage.beforeYouStart.line.1") <a href=@{appConfig.customsDeclarationsGoodsTakenOutOfEuUrl}>@messages("startPage.beforeYouStart.line.1.link")</a>
-      </p>
-      <p id="before-you-start-element-2">@messages("startPage.beforeYouStart.line.2")</p>
-      <p id="before-you-start-element-3">@messages("startPage.beforeYouStart.line.3")</p>
-
-      <h2 id="information-you-need" class="heading-medium">@messages("startPage.informationYouNeed.header")</h2>
-      <p id="information-you-need-element-1">@messages("startPage.informationYouNeed.line.1")</p>
-      <ul id="information-you-need-list" class="list list-bullet">
+    @sectionHeader(messages("startPage.title.sectionHeader"))
+
+    @pageTitle(messages("startPage.title"))
+
+    <p class="govuk-body-l govuk-!-margin-bottom-8"> @messages("startPage.description") </p>
+
+    <hr class="govuk-section-break govuk-section-break--xl govuk-section-break--visible">
+
+    <h2 id="contents" class="govuk-heading-m">@messages("startPage.contents.header")</h2>
+    <ol class="govuk-list">
+        <li> - <a class="govuk-link" href="#before-you-start">@messages("startPage.beforeYouStart.header")</a></li>
+        <li> - <a class="govuk-link" href="#information-you-need">@messages("startPage.informationYouNeed.header")</a></li>
+        <li> - <a class="govuk-link" href="#report-your-arrival-and-departure">@messages("startPage.reportYourArrivalAndDeparture.header")</a></li>
```

Comment:
Should we create a class to add this dash automatically to the list?

Reply:
We are still able to use this by providing 
```
govuk-list__item--dashed:before {
  content: "-"
}
```
and add correct class to every `li`

Reply:
Or as later
```
govuk-list--dashed li:before {
  content: "-"
}
```

Reply:
GDS Design System does not have dash list anymore.
But I have made that part of a CSS class. Thank you!